ApiBundle/Tests/Transformer/ContentTransformerTest.php

```diff
@@ -31,24 +34,38 @@ class ContentTransformerTest extends \PHPUnit_Framework_TestCase
      */
     public function setUp()
     {
-        $this->eventDispatcher = Phake::mock('Symfony\Component\EventDispatcher\EventDispatcherInterface');
-
         $this->content = Phake::mock('OpenOrchestra\ModelInterface\Model\ContentInterface');
+        Phake::when($this->content)->getId()->thenReturn(1);
         $this->status = Phake::mock('OpenOrchestra\ModelInterface\Model\StatusInterface');
-        $this->statusId = 'StatusId';
-        Phake::when($this->status)->getId(Phake::anyParameters())->thenReturn($this->statusId);
+        $this->contentData = 'contentData';
+        $this->contentDate = '2015/09/01';
+        $this->contentBool = true;
+        Phake::when($this->content)->getContentId(Phake::anyParameters())->thenReturn($this->contentData);
```

Comment:
Those methods are only getters, we can assume that they would not receive a parameter, so the Phake::anyParameters() is useless in this case